onnx/test/shape_inference_test.py

```diff
@@ -173,6 +194,34 @@ def _compare_value_infos(
                 "Unrecognized value info type in _compare_value_infos: ", str(vi_type)
             )
 
+    def _test_an_op(self, op_name, test_function_name) -> None:
```

Comment:
Naming: _test_op, _test_single_op etc. ?

Reply:
Changed the function to a decorator named `single_op_all_version_tester`.

Reply:
The decorator is removed.